tests/res/enkf/test_row_scaling_case.py

```diff
@@ -370,3 +383,82 @@ def test_row_scaling_using_assign_vector(self):
+            update_fs1 = main.getEnkfFsManager().getFileSystem("target1")
+
+            # The first smoother update without row scaling
+            es_update = ESUpdate(main)
+            run_context = ErtRunContext.ensemble_smoother_update(init_fs, update_fs1)
+            rng = main.rng()
+            es_update.smootherUpdate(run_context)
+
+            # Configure the local updates
+            local_config = main.getLocalConfig()
+            local_config.clear()
+            obs = local_config.createObsdata("OBSSET_LOCAL")
+            obs.addNode("WBHP0")
+
+            ministep1 = local_config.createMinistep("MINISTEP1")
+            local_data1 = local_config.createDataset("LOCAL1")
+            local_data1.addNode("PORO")
+            row_scaling1 = local_data1.row_scaling("PORO")
+            ministep1.attachDataset(local_data1)
+            ministep1.attachObsset(obs)
+
+            ministep2 = local_config.createMinistep("MINISTEP2")
+            local_data2 = local_config.createDataset("LOCAL2")
+            local_data2.addNode("PORO")
+            row_scaling2 = local_data2.row_scaling("PORO")
+            ministep2.attachDataset(local_data2)
+            ministep2.attachObsset(obs)
+
+            updatestep = local_config.getUpdatestep()
+            updatestep.attachMinistep(ministep1)
+            updatestep.attachMinistep(ministep2)
+
+            # Apply the row scaling
+            ens_config = main.ensembleConfig()
+            poro_config = ens_config["PORO"]
+            field_config = poro_config.getFieldModelConfig()
+            grid = main.eclConfig().getGrid()
+
+            row_scaling1.assign(field_config.get_data_size(), SelectLayer(0, grid))
+            row_scaling2.assign(field_config.get_data_size(), SelectLayer(1, grid))
+
+            update_fs2 = main.getEnkfFsManager().getFileSystem("target2")
+            es_update = ESUpdate(main)
+            run_context = ErtRunContext.ensemble_smoother_update(init_fs, update_fs2)
+            es_update.smootherUpdate(run_context)
+
+            init_node = EnkfNode(poro_config)
+            node1 = EnkfNode(poro_config)
+            node2 = EnkfNode(poro_config)
+            for iens in range(main.getEnsembleSize()):
+                node_id = NodeId(0, iens)
+
+                init_node.load(init_fs, node_id)
+                node1.load(update_fs1, node_id)
+                node2.load(update_fs2, node_id)
+
+                init_field = init_node.asField()
+                field1 = node1.asField()
+                field2 = node2.asField()
+                for iv, v in zip(init_field, field2):
+                    assert iv != v
+
+                    # If it were not for the random sampling in the
+                    # obs_data_allocE() function called once per ministep in
+                    # enkf_main_analysis_update() - we should have had this
+                    # assert:
+                    #
+                    # assert v1 == v2
```

Comment:
```suggestion
```

Maybe add it to the top comment as a NOTE

Reply:
Essentially the same information is already present at the top - I have just removed it.